CMakeLists.txt

```diff
@@ -293,6 +293,13 @@ endif()
 # OpenMP (optional)
 option(WITH_OPENMP "Build with parallelization using OpenMP" TRUE)
 if(WITH_OPENMP)
+  if(APPLE)
```

Comment:
Can you remove this change from this PR?

Reply:
If this is removed, SuiteSparse can't find OpenMP, and I've moved this to #6114 , but there are still some issues and details to be discussed. Merge this after the #6114 merge?

Reply:
What if you try remove the local findOpenMP and use the one from Cmake itself, as in #6100?